src/Employer/Employer.Web/Orchestrators/Part2/QualificationsOrchestrator.cs

```diff
@@ -154,6 +178,16 @@ private void SyncErrorsAndModel(IList<EntityValidationError> errors, Qualificati
                 .ForEach(r => errors.Remove(r));
         }
 
+        private int? SanitizeEditIndex(int? editIndex, IEnumerable<Qualification> vacancyQualifications)
+        {
+            if (editIndex.HasValue && 
```

Comment:
This can be simplified to `editIndex >= 0 && editIndex < vQ.Count()`

Reply:
Also if we have explicit edit endpoint then the index will be required

Reply:
👍